sway/commands/bar/strip_workspace_name.c

```diff
@@ -0,0 +1,29 @@
+#include <string.h>
+#include <strings.h>
+#include "sway/commands.h"
+#include "log.h"
+
+struct cmd_results *bar_cmd_strip_workspace_name(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc,
+				"strip_workspace_name", EXPECTED_EQUAL_TO, 1))) {
+		return error;
+	}
+	if (!config->current_bar) {
+		return cmd_results_new(CMD_FAILURE,
+				"strip_workspace_name", "No bar defined.");
+	}
+	if (strcasecmp("yes", argv[0]) == 0) {
```

Comment:
Copied this part from `strip_workspace_number.c`. Might be better to use `parse_boolean` here.

Reply:
+1, suggest using it in both places